### DISCUSSION TITLE
How can I make a certain window always be floating on whim startup

### DISCUSSION THREAD

**Discussion by Minoo7** [dalyIsaac/Whim#878] [Q&A] [answered]
Is it possible to configure the Whim config to make a window of a certain title or exe always be floating, like in FancyWM?

Thanks

**Comment by urob** [comment, marked as answer]
This can be done using [`Filtering` rules](https://dalyisaac.github.io/Whim/docs/customize/filtering.html)

**Reply by Minoo7** [reply]
Thank you this did work.
Would love for this to be only on startup/initial and that it would be possible to toggle it to stop floating with the toggle float, like in FancyWM. But I guess Whim ignores the window completely and functions like `toggle float` can't be called upon it. I guess what I am essentially after is for a window to be marked as floating on startup/initially
